aws.yml

```diff
@@ -0,0 +1,25 @@
+---
+stage_keypair_name: "osx_rsa"
+stage_ami: "ami-d2414e38"
+stage_instance_type: "t2.micro"
+stage_region: "eu-west-1"
+stage_subnet_id: "subnet-2c67fe64"
+stage_security_groups: ["sg-7b78fe07"]
+stage_associate_public_ip: true
+stage_access_key_id: ""
```

Comment:
not needed

Reply:
security_groups and subnet_id are different between users, no need to overwrite them each pull